_data/publishers.csv

```diff
@@ -1101,7 +1101,6 @@ http://unifiedjournals.org/,Unified Journals,,
 http://www.ujconline.net/,Unique Journals Communication,UJC,
 http://www.uniqueresearchjournals.com/index.htm,Unique Research Journals,,
 http://www.unitedscholars.net/home.html,United Scholars Publications,,
-http://unitedscientificgroup.com/,United Scientific Group,,
```

Comment:
You have wrongly included this in the list

Reply:
Please provide some evidence that the group is wrongly included in the list. 
Responding with evidence to the contrary on the points listed here will suffice: <https://predatoryjournals.com/about/#some-basic-criteria>